Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.h

```diff
@@ -118,6 +118,9 @@ class NetworkRTCProvider : private FunctionDispatcher, private IPC::MessageRecei
     IPC::Connection& connection() { return m_ipcConnection.get(); }
     Ref<IPC::Connection> protectedConnection() { return m_ipcConnection.get(); }
 
+    RefPtr<NetworkConnectionToWebProcess> protectedConnectionToWebProcess();
```

Comment:
This getter does not seem to be used in this patch.

Reply:
Oh weird. Will remove.